### ISSUE TITLE
Review "how it works" content

### ISSUE THREAD

**Issue by JoannaHill** [openaustralia/planningalerts#1715]
How it works section on the landing page is taken from service design mock ups and needs a content review. Access current visual design in Figma here: https://www.figma.com/file/GEaOuajueHGvfaFPsOUZ9n/Application-detail-page?type=design&node-id=0%3A1&t=VvhYlCbZ1yYos7y1-1

**Comment by mlandauer** [comment]
The current wording in Figma is:
1. Search: Search for an address you're interested in using the search bar
3. Get Notified: Sign up to receive email notifications about future developments in your email
4. Comment: Look through the applications that interest you and comment on them. Comments get sent directly to council and are posted publically on our site

**Comment by mlandauer** [comment]
What I think is good:
* The order matches what most people experience.
* It shows that you need to search first before signing up for an alert
* Use of the word "notifications" rather than alerts because that is now more common
* That it shows that commenting is a natural next step and you learn that at the start

Things that are less good:
* In 1. saying "search bar". We don't have a search bar. We have a "search applications" menu item in the nav and we have a big box on the landing page. Maybe just remove the words "using the search bar"?
* In 1. we're not searching for an address we are searching for applications as is reflected by the new wording of the search box
* In 2. using the word "email" twice. Should this be "Sign up to receive email notifications about future developments"
* In 3. "our site" is confusing and not user centered strangely. Probably best just to name it.

Possible new wording:

1. Search: Search for applications in an area you're interested in
2. Get Notified: Sign up to receive email notification about future developments in your search area
3. Comment: Look through the applications that interest you and comment on them. Comments get sent directly to council and are posted publically on Planning Alerts.

On a different note this "how it works" ignores the process of us collecting the information from council websites. Would this be an oppurtunity to explain that in the simplest possible way so that people fundamentally understand the limitations of the site and that we are dependent on council for the data and that we ourselves are not the council?

**Comment by mlandauer** [comment]
I just noticed that you have a bit of explanatory text above the 1.2.3 that explains the scraping (taken from the about page):
"PlanningAlerts is a free service which searches as many planning authority websites (Councils or state authorities) as it can find. Then it emails you details of applications near you."

Maybe just focus on the scraping side and leave the user side to the 1.2.3. box. For instance:
"PlanningAlerts collects information from as many local councils and state planning authorities as it can so then you can..."

**Comment by mlandauer** [comment]
@katska I'm keen to get your thoughts on this please

**Comment by katska** [comment]
We're looking at this:
<img width="918" alt="Screenshot 2023-05-24 at 10 25 48 am" src="https://github.com/openaustralia/planningalerts/assets/1001129/09213235-6fb2-484e-b309-73a97114327e">


@mlandauer 
To be more specific If the words are a 'how to' or 'what steps do we facilitate' then the words you use are close, as they say something about each main action. If they're meant to convey something of the big picture then it's a bit of a different problem. 

First - clarifying what I see as the goal here: to provide a simple overview of the service, make it clear how the whole thing works and how people can use it in their daily lives?

**Comment by katska** [comment]
If this section is an overview and introduces the main concepts, then I wonder if it's the right place for specific instructions like 'using the search bar'. That feels like the kind of detail given in a step by step instruction.

With that in mind I see the core of an overview in three steps being something more like the existing language of the site. 

1 - **Search** for development applications near you
2 - **Get alerts** Sign up and get free email alerts whenever new developments are proposed
3 - **Make a public comment** when something that matters to you


I'm not sure how to talk about all this, and I'm trying my best here to unpick what's going on for me.

I'll try and talk through my thinking here around one small piece. One principle that comes in here is 'never using a long word where a short word will do', which is a big part of how OpenAustralia uses language. 

I'm sticking with the word alert for now, partly I feel resistance to the words notify and notification. One tiny point worth digging into, for me. I have heard people suggest recently that other people use the word notification more but I don't know if that's true, in what context it's true, and I also don't know whether that by itself is a good reason to change that word. We often try to lead rather than follow, and in doing this there will be tension between a more complicated word has become common, versus a clearer simpler word that has fallen or been pushed out of use. Also to me notify sets a passive expectation, it says 'this thing is happening', alert on the other hand says 'wake up' ... to this thing that's happening, be responsive. And lastly, the service is called Planning Alerts, not Planning Notifications. 

In short, we want people to understand the site easily and at the same time not be drawn into overly complex bureaucratic language just because people are used to it. On the other hand if we've clearly learned that no-one understands that word then we could change it, and embrace that change, I simply haven't seen that evidence.

**Comment by katska** [comment]
Other notes to consider in iterating steps
Search is the action "Find" is the result. Still here is a mix of directions  of steps to take, and then things that will happen as a result. 
'near you' is a concept that implies some of this. Is it enough? 
Is introducing the idea that it's an address based search important? 
It's not a search for an address, but a search for applications within the vicinity of a specific street address.

The steps miss out looking through applications that interest you. I like where that's going, will think about how best to incorporate.

**Comment by katska** [comment]
Looking at "PlanningAlerts is a free service which searches as many planning authority websites (Councils or
state authorities) as it can find. Then it emails you details of applications near you"

This approach combines the explanation of how the service operates along side signing up. It might be more effective to focus on one aspect at a time, and prioritise the main flow of using the site/service.

To emphasize the main flow for signing up, we could first summarize the search and sign up steps, and normalize the idea of commenting later.

_Easily search for development applications near you, receive free email alerts for new developments, and have the opportunity to make public comments._

Discussion Point:
The assumptions to embed include the right to know about planned developments, the influence one can have by taking action and expressing opinions, and contributing to community improvement.

Proposed Content:
_Easily search for development applications near you, receive free email alerts for new developments, and make public comments on projects that could impact you and your community._

Discussion Point:
The current wording seems passive and implies that comments are resistance to inevitable development. Through our research discussions we uncovered the importance of revealing that later intent. That's not only about 'having a say', a problematic phrase, we can begin to expand into the idea of people coming together to actively shape the community's future.

Proposed Content:
_Easily search for development applications near you, sign up for free email alerts for new plans, and actively participate by making public comments to help shape your community's future._

Discussion Point:
The description could address how the service gathers information and set expectations about reliability, limitations, and the organisation behind it.

Proposed Content:
_Easily search for development applications near you, receive free email alerts for new plans, and actively participate by making public comments to help shape your community's future._ 

_We do our best to gather and provide reliable information from Planning Authorities all around Australia._

**Comment by katska** [comment]
@mlandauer @JoannaHill / @sabpop I've done my best to have a go at discussing the words you wanted my input on. Does any of this help?  I welcome all your thoughts on where this might go next, what you think we can and should include.

**Comment by katska** [comment]
Coming back to this - more thoughts on wording, getting hung up on the nuances of alerts vs notifications I forgot tobe encouraging. So here's another go, bringing back Have your say and leaving out alert. for now. Planning authority has come up as problematic lately and I'm wondering if 'council' could work as a default and we mention other planning authorities in detailed info later.


1 - Pop in your address: You start by entering your address into the search bar. 
2 - Get notified: Once your address is in, we start scanning new planning applications around your locale. If something pops up, you'll get a friendly email letting you know.
3 - Have your say: Now, it's your turn. Chime in with comments on an application that matters to you. We send the comment on to council where the application's up for discussion. 

Remember that these applications can have a significant impact on your local area, including changes to infrastructure, building projects, and more. By using PlanningAlerts.org.au, you're taking an active role in the development and future of your community.

Your voice matters in your community's development.
